Makefile

```diff
@@ -73,6 +73,7 @@ str_match_test:
 # testing
 ##
 test: joy joy_test.py
+	./download_test_pcaps.sh test_pcaps/pcapr
```

Comment:
Yeah, we should avoid shell script for windows compatibility. Python would be a better option here. Also, do these caps change a lot? The reason I ask, is if they are static and used for testing, then why not just commit them into the resources directory?